README.md

```diff
@@ -1,2 +1,316 @@
+
+## <i class='bx bx-desktop bx-burst' style='color:#f7efdb'  ></i> Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git or Github Desktop
+
+- Code editor like: [Visual Studio code](https://code.visualstudio.com/), [Sublime text](https://www.sublimetext.com/) or others...
+
+- Npm and Node.js
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+- You can download the **Zip** file on the GitHub repository, or clone the repository with:
+```
+$git clone https://github.com/elafildecolor/MyPortfolio.git
+```
+
+- Access the cloned directory with:
+```
+$cd MyPortfolio
+```
+
+- Open it with your favorite code editor or with the live server
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+To install linters and other dependencies run:
+```
+$npm install <dependency>
+```
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To analyze the source code in your local environment with linter follow these steps:
+
+- Looking for linter errors in HTML files run:
+```
+$npx hint .
+```
+
+- Looking for linter errors in CSS | SASS files run:
+```
+$npx stylelint "**/*.{css,scss}"
+```
+
+- Looking for linter errors in JavaScript files run:
+```
+$npx eslint .
+```
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+<!--
+### Deployment
+
+You can deploy this project using:
+
+
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## <i class='bx bx-id-card bx-fade-left' style='color:#a4a4a4' ></i> Authors <a id="authors"></a>
+
+<i class='bx bxs-user-badge bx-tada' style='color:#ace1f3' ></i> **Paul Sáenz Sucre**
+
+- GitHub: [@elafildecolor](https://github.com/elafildecolor) <i class='bx bxl-github bx-tada' style='color:#efefef' ></i>
+- Twitter: [@paulsaenzsucre](https://twitter.com/paulsaenzsucre) <i class='bx bxl-twitter bx-tada' style='color:#44d7ff' ></i>
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paulsaenzsucre) <i class='bx bxl-linkedin bx-tada' style='color:#278ace'  ></i>
+<!--
+<i class='bx bxs-user-badge bx-tada' style='color:#ace1f3' ></i> 
+
+
+**Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle) <i class='bx bxl-github bx-tada' style='color:#efefef' ></i>
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle) <i class='bx bxl-twitter bx-tada' style='color:#44d7ff' ></i>
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) <i class='bx bxl-linkedin bx-tada' style='color:#278ace'  ></i>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+-->
+<!-- FUTURE FEATURES -->
```

Comment:
- [ ] I think it would be good if you could remove all the content that is commented on in your README file. It does not appear on the render but It makes your README look good 😃 😎